includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -428,7 +428,7 @@ public function pull( $items ) {
 				update_post_meta( $new_post, 'dt_full_connection', true );
 			}
 
-			if ( ! empty( $post_array['meta'] ) ) {
+			if ( isset( $post_array['meta'] ) && is_array( $post_array['meta'] ) ) {
```

Comment:
The empty check was preventing the hook from running if there was no meta because `empty( [] ) === true`. My understanding is that the hook should fire regardless of whether there is or is not meta so I've changed it to fire when the array is set.